doc/nest-09.md

```diff
@@ -0,0 +1,55 @@
+# Managing Nest's repositories
+
+Several repositories are available for Nest. The official ones are `stable`, `beta` and `nightly`.
```

Comment:
Those explanations about repositories should be placed way earlier in the tutorial. They also apply to the `training-wheels` mode.